examples/realworld/main.wasp

```diff
@@ -25,7 +25,8 @@ app Conduit {
     ("moment", "2.29.1"),
     ("@material-ui/core", "4.11.3"),
     ("@material-ui/icons", "4.11.2"),
-    ("slug", "4.0.2")
+    ("slug", "4.0.2"),
+    ("lodash", "^4.17.21")
```

Comment:
It seems that `lodash` was one of the deps that were installed by Create React App and we need to explicitly add it now.

Reply:
While we're at it, can we try to do [this](https://github.com/wasp-lang/wasp/pull/747#discussion_r996232126) too (we're only using `lodash` to get its `merge` function, if I'm not mistaken)?

Reply:
I've gone with using `lodash.merge` package since `merge` is a more complex function then just recursivelly applying `Object.assign` (it handles arrays as well for example) which gives the most freedom in the future.

I think this gives us an okay combo: smaller dep install and most power.

Reply:
@infomiho Great, `lodash.merge` is perfectly fine (even more than fine, it was precisely what the linked comment suggested :))

Hey @Zeko369, nice to see you here!

> Technically in this example lodash is only used for `_.take` and `_.sortBy` functions, which can be done with `Array.prototype.slice` and `Array.prototype.sort` so it can technically be removed and replaced with native JS functions

This is true, yes. _Realworld_ is just an example app so it isn't that big of a deal, but Miho can replace it if he finds the time.

Reply:
@Zeko369 That's a valid suggestion, I would maybe keep it like this in this PR and update the example up code in the next one :)

If you are up for it, you can do it as well? Let me know if I can help you with the set up.